src/plugins/tabs/_screen-sm-max.scss

```diff
@@ -9,18 +9,29 @@
 %tabs-screen-sm-max-details {
 	> {
 		details {
-			border: 1px solid #adadad;
-			border-top: 0;
+			border: {
+				color: $tablist-screen-sm-details-border-color;
+				style: $tablist-screen-sm-details-border-style;
+				width: $tablist-screen-sm-details-border-width;
+			}
 			padding: 6px 12px;
 
 			&:first-of-type {
-				border-top-left-radius: 4px;
-				border-top-right-radius: 4px;
+				border: {
+					top: {
+						left-radius: $tablist-screen-sm-details-border-radius;
+						right-radius: $tablist-screen-sm-details-border-radius;
+					}
+				}
 			}
 
 			&:last-of-type {
-				border-bottom-left-radius: 4px;
-				border-bottom-right-radius: 4px;
+				border: {
+					bottom: {
+						left-radius: $tablist-screen-sm-details-border-radius;
+						right-radius: $tablist-screen-sm-details-border-radius;
```

Comment:
We should investigate if we can remove the bottom so that this can be applied to all edges